.ruby-version

```diff
@@ -1 +1 @@
-2.3.1
+2.3.3
```

Comment:
Sure, the warnings are annoying, but most of them have open PRs so should be fixed soon. The logistics-dashboard is running 2.4.0 as of today, so we can use that as a testbed.